scripts/process-reboot-cause

```
@@ -67,24 +70,23 @@ def read_reboot_cause_files_and_save_to_db(device='npu'):
                     if device == 'npu':
                         _hash = '{}|{}'.format(REBOOT_CAUSE_TABLE_NAME, data['gen_time'])
                     else:
-                        # Ensure keys exist
                         if 'name' not in data:
                             sonic_logger.log_warning(f"Missing 'name' in reboot-cause file")
-                            continue  # Skip this file
+                            continue
                         _hash = f"{REBOOT_CAUSE_TABLE_NAME}|{device.upper()}|{data['name']}"
                     db.set(table, _hash, 'cause', data.get('cause', ''))
                     db.set(table, _hash, 'time', data.get('time', ''))
                     db.set(table, _hash, 'user', data.get('user', ''))
                     db.set(table, _hash, 'comment', data.get('comment', ''))
                 except json.decoder.JSONDecodeError as je:
-                    sonic_logger.log_error("Unable to process reload cause file {}: {}".format(x, je))
-                    pass
+                    sonic_logger.log_error(f"Unable to process reload cause file {x}: {je}")
```

Do we want to throw this error for the prev_reboot_time.txt file always?